templates/designatebackendbind9/config-named/options.conf

```diff
@@ -0,0 +1,38 @@
+options {
+        allow-new-zones yes;
+        directory "/var/named-persistent";
+
+        # The default for named is to listen on everything and it's controlled
+        # by different settings for different IP versions. If the objective is to
+        # restrict to listening on a given address, we need to explicitly turn
+        # listening *off* for the different IP version.
+
+        # TODO: The '*'s need to be replaced by actual addresses.
+{{ if eq .IPVersion "4" }}
+        listen-on port 53 { *; };
+        listen-on-v6 { none; };
```

Comment:
Are these mutually exclusive on OSP18? I am looking at my named.conf file on my devstack and I see
listen-on port 53 { 192.168.124.114; };
listen-on-v6 port 53 { ::1; };

Reply:
I also see this https://opendev.org/openstack/designate/src/branch/master/devstack/designate_plugins/backend-bind9#L167

Reply:
We need the predictable IPs to set these to what they are going to be.

Reply:
Sorry, I misunderstood the question. Yes, there are mutually exclusive. I don't think dualstack support is on the road map for now.